### PR TITLE
feat: step-guide

### PR DESCRIPTION
**Pull request by prathmeshkhandelwal1**
- Added steps guide for deploying `xERC20` and `Lockbox`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,6 +85,14 @@ yarn coverage
 
 Configure the `.env` variables.
 
+> :information*source: Notice: \_Please ensure that all private keys (PKs) are prefixed with `0x` before using or submitting them. This prefix is necessary for proper formatting and compatibility.*
```
Comment (0xShaito) [inline README.md]: ty :pray:

File: example.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
+## Setup Guide to Deploy an xERC20 Token
+
+This guide provides a detailed, step-by-step process to deploy an xERC20 token using this repository. We will first demonstrate how to deploy the xERC20 token alone, and then we’ll cover the scenario in which you want to deploy both the xERC20 token and the lockbox.
```
Comment (0xShaito) [inline example.md]: We will not be covering the other case right? The deploy with lockbox I mean

File: example.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
+## Setup Guide to Deploy an xERC20 Token
```
Comment (0xShaito) [inline example.md]: A part of this is written in the README. Can we merge it with that and consolidate in there so it shows when you open the repo?

Also I think it's a good idea to move the section now to the top of the README as it's in the bottom now!

**Review by 0xShaito — CHANGES_REQUESTED** [review]

**Review by 0xShaito — APPROVED** [review]

**Review by rhlsthrm — APPROVED** [review]

**Review by 0xGorilla — APPROVED** [review]

**Review by 0xGorilla — CHANGES_REQUESTED** [review]
Please pull and push. All of the broadcast changes were already merged in another PR

**Review by 0xGorilla — APPROVED** [review]

**Pull request merged**